app/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,9 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
 
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```

Comment:
Question: Can you explain what this permissions do and why they are necessary? (Even though it seems clear I want to be sure)

Reply:
Of course. In essence, coarse location is permission to access location, but *approximative* (the precision can be terrible, it's just to get an idea of where the user is).

Fine location is more precise, but poses more problems in terms of privacy. Hence, Android guidelines tell us to, if possible, accept that the user only grants coarse location.

Since our app only displays the placement of the user of the map, I estimated it wasn't important to know the exact position of the user, so I implemented it in a way that accepts both coarse and fine location.

Reply:
Following what you are telling me I'll accept it in Sonar since it seems essential for the app's functionality.